.github/workflows/github-action-scan.yml

```diff
@@ -3,27 +3,16 @@ name: Build SecHub GHA (scan)
 
 on:
   push:
-    branches:
-      - main
-      - master
-      - develop
-      - hotfix
+    paths:
+      - 'github-actions/**'
```

Comment:
If we ever add any other action to `github-actions` directory this workflow will run for that action as well
probably won't happen but maybe it's better to set it to `github-actions/scan/**` (just to be sure in the future)

Reply:
Hmm. I thought that if we have more Actions in the future, we would have one workflow for them all.
(Maybe rename this one...)

Reply:
I don't think that makes sense.

If you take a look at the at the workflow file you'll see that it's actually tailored to the source code of the `github-actions/scan` directory.

I'd rather stick to my suggestion from above

Reply:
full path and name added